### ISSUE TITLE
More config options?

### ISSUE THREAD

**Issue by deebiia** [Tiviacz1337/Travelers-Backpack#991]
A slight "issue" I have with the dragon backpack is that the regen timer is like 2 seconds slower than the other effects on the backpack. It's mildly annoying as it flashes. Also, the dragon backpack really doesn't need the water buff. The regen, fire prot, and strength effects are enough.

I would like to see more config options available. Such as effect duration (in seconds), cooldown timer config, and the ability to adjust the backpack effects as needed. I saw a previous post asking for the ability to change the effects and the reply was simply, "not possible". I'm not the most skilled at making mods or altering them to fit my server better, but when looking into the Abilities to alter them, there is just nothing to modify to make any adjustments.

TL;DR
- More config options would be nice, or just the ability to modify the files myself to fit my server better.
- The night vision-water buff on the dragon backpack could be removed. It doesn't really fit and is a little much.
- The dragon backpack regen is like 2 seconds shorter than the other buffs and is slightly annoying to see flashing as it "wears off".
- Perhaps allow the bee backpack to trigger on ranged attacks as well? And increase the duration of the effect a bit?

**Comment by Tiviacz1337** [comment]
Abilities changes/overhaul is planned, upgrades updates will come first though

**Comment by Tiviacz1337** [comment]
Added this suggestion.
The Backpack effects can be removed (from exsisting backpacks) or added to any backpack
Cooldowns can be configured for abilities that add cooldown (like creeper backpack)
Effect amplifier can be modifed 
As for now only available for 1.20.1 Forge and Fabric, 1.21.1 Forge, NeoForge, Fabric. 1.21.4 coming soon

https://www.curseforge.com/minecraft/mc-mods/travelers-backpack
Versions that add this feature:
Forge 1.20.1 - 9.1.28+
Fabric 1.20.1 - 9.1.25+
Forge 1.21.1 - 10.1.10+
NeoForge 1.21.1 - 10.1.15+
Fabric 1.21.1 - 10.1.13+
